chromadb/api/segment.py

```diff
@@ -806,7 +807,7 @@ def _validate_embedding_record_set(
         """Validate the dimension of an embedding record before submitting it to the system."""
         add_attributes_to_current_span({"collection_id": str(collection["id"])})
         for record in records:
-            if record["embedding"]:
+            if record["embedding"] is not None:
```

Comment:
similar question to somewhere else, why is it possible for this to be None now?

Reply:
It was always possible for this to be `None`.  For example, in `_records` within `chromadb/api/segment.py`, the embeddings can be set to `None`. The added check is because you can check the existence of a python list by checking the truth value of it, but you cannot with a numpy array. A ValueError is thrown when trying to do so.